laravel/app/Console/Commands/UpdateAnesthesiologistsData.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Anesthesiologist;
+use Illuminate\Console\Command;
+use Illuminate\Support\Carbon;
+
+class UpdateAnesthesiologistsData extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'update:anesthesiologists_data';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Update the anesthesiologists table with information about who is working today';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $json = file_get_contents('https://remodel.osuanes.com/QgendaSchedule/');
+        $json_data = json_decode($json, true);
+
+        //get scheduling date in format: "yyyy-mm-ddT00:00:00"
+        $date = substr(date("c", strtotime('+2 day')), 0, -14) . "00:00:00";
+        if(date("l", strtotime('today')) == 'Thursday' || date("l", strtotime('today')) == 'Friday'){
+            $date = substr(date("c", strtotime('+4 day')), 0, -14) . "00:00:00";
+        } elseif (date("l", strtotime('today'))=='Saturday') {
+            $date = substr(date("c", strtotime('+3 day')), 0, -14) . "00:00:00";
+        }
```

Comment:
I'm confused by this. Why is date parsing is different based on today's date?

Reply:
It is related to REMODEL's scheduling logic. Residents schedule for +2 days, but only schedule for week days (Mon-Fri).
So on Mondays schedule for Wed, it follows like this below 
Mon -> Wed (+2days)
Tues -> Thurs (+2days)
Wed -> Friday (+2days)
Thurs -> Monday (+4days)
Fri -> Tues (+4days)
Sat -> Tues (+3days)
Sun -> Tues (+2days)